Feat/liste beer

TheNoobProgrammeur opened the pull request:
Ajout du listing des bière et leur notation.
J'ai fait assez brut. Il y a sans doute des choses a clean ou à faire plus propre.
Mais on a les fonctionnalités.

Commit:
Liste des biéres

Commit:
✨ Ajout de la notation de biére

BBR2394 commented on app/src/main/AndroidManifest.xml:
```diff
@@ -9,10 +9,16 @@
+        <activity
+            android:name=".activity.DetailBeer"
+            android:label="@string/title_activity_detail_beer"
+            android:theme="@style/AppTheme.NoActionBar"></activity>
```
pour le style, ce n'est pas mieux de le définir dans le xml de l'activity, que dans le manifest d'android ? (perso je ne sais pas ce qui est mieux ^^)

BBR2394 reviewed (approved):
juste un commentaire pour le @style dans le manifest (apres si ca ne pose pas de probleme ^^)

Commit:
Merge branch 'master' into feat/listeBeer

Commit:
:lipstick: Update darkmode

Commit:
:lipstick: Update style

BBR2394 committed:
:lipstick: fix color bottom nav bar for dark mode

as  i said, it was not so easy to set it because i have created a theme specifically for this component. The component get his background color from the theme, but because it is the same theme than the background, it is the same color, we dont want that, so i apply another theme for only the navigation bar, whith the background from the theme, and aafter it selectes the corect color from the styles file

acroquelois committed:
:lipstick: Refacto add beer

acroquelois committed:
:bug: Fix Adding beer firebase

acroquelois committed:
:lipstick: Refacto beer list & beer detail

acroquelois committed:
:sparkles: Add average on adapter

acroquelois committed:
:bug: fix form decimal on beer degree

acroquelois committed:
:ambulance: Switch firebase

Commit:
💚 add fix TU

Commit:
💚 add fix TU Two
